style.css

```diff
@@ -96,3 +105,188 @@ header {
   width: 96vw;
   top: 0;
 }
+
+/* work section */
+
+.work-section {
+  display: grid;
+  grid-template-columns: auto;
+}
+
+.work-section-title {
+  color: var(--heading-color);
+  font-size: var(--heading-font-size);
+  max-width: 60%;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+}
+
+.work-section-title > span {
+  width: 80px;
+  height: 6px;
+  background-color: #43b886;
+  margin: 0 auto;
+  border-radius: 3px 3px;
+}
+
+/* cards */
+
+.card-container {
+  margin: 0 auto;
+  width: 100%;
+}
+
+.img-placeholder {
+  height: 500px;
+  border-radius: 8% 8% 0 0;
+}
+
+/* work section cards */
+.card-work {
+  max-width: 23rem;
+  background-color: #ebf0ee;
+  height: 554px;
+  margin: 0 auto;
+  margin-top: 10%;
+  display: flex;
+  width: 75%;
+  flex-direction: column;
+  border: solid #d0d9d4 1px;
+  border-radius: 2% 2% 0 0;
+}
+
+h2 {
+  color: #172b4d;
+}
+
```

Comment:
- [ ] Please stick with the margin and width provided in  [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) template 


![image](https://user-images.githubusercontent.com/74744787/159041513-7d80bb88-71ec-4e53-8171-ef2cf4f4b57f.png)